source/about/self-hosted-subscriptions

```diff
@@ -0,0 +1,172 @@
+====================
```

Comment:
The file extension isn't yet defined for this new page as .rst
Also - will we want this new page surfaced up in the LHS?

Reply:
.rst extension added and yes, let's add to LHS